Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/Utilities/Extensions.cs

```diff
@@ -0,0 +1,244 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should this be in Microsoft.Toolkit.Uwp?

Reply:
These are all internal and unless we are ready to make them public, they should remain where they are.